### DISCUSSION TITLE
how to add jars for scala kernel

### DISCUSSION THREAD

**Discussion by git0114106** [microsoft/vscode-jupyter#10151] [Questions and Answers] [answered]
how do I add jars to my kernel before it launches?.
Apart from standard libraries, the notebook is not picking up the ones in CLASSPATH
import org.apache.commons.io._ is giving me error 
cmd0.sc:1: object apache is not a member of package org

**Comment by DonJayamanne** [comment, marked as answer]
You can add custom env variables to your kernels as follows:
* Create a file named `.env` in your workspace folder
* Add an entry as follows
```
CLASSPATH=XYZ
```
* Reload VS Code and run the kernel
* The env variable `CLASSPATH` will be available to the kernel
Hope this helps.
